WooCommerce/Classes/POS/Controllers/PointOfSaleItemsController.swift

```diff
@@ -21,9 +21,16 @@ protocol PointOfSaleItemsControllerProtocol {
     func refreshItems(base: ItemListBaseItem) async
     /// Loads the next page of items for a given base item.
     func loadNextItems(base: ItemListBaseItem) async
+    /// Enables coupons in store settings, if needed
+    func enableCoupons() async
 }
 
-
+@available(iOS 17.0, *)
+extension PointOfSaleItemsControllerProtocol {
```

Comment:
Do we need this extension with default implementation?

Reply:
We do unfortunately, since we need to call `couponsController.enableCoupons()` in the aggregate, and this is a specialization of couponsController `PointOfSaleItemsControllerProtocol`